### PR TITLE
TASK: Add circle ci config

### PR DESCRIPTION
**Pull request by johannessteu**

### CODE REVIEW

**Comment by johannessteu** [comment]
Just for the records - the current failing e2e tests:
https://gist.github.com/johannessteu/2cd198b2cc54decddec59550fc80301e

Dunno if it's about the setup or the tests are just broken atm

**Comment by johannessteu** [comment]
Anything i can do/help with to get this pushed?

**Comment by dimaip** [comment]
Doh, sorry for not getting back to it, really hard to keep track of everything that's going on lately here... Will try to find the time to give it a proper test on Friday.

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,118 @@
+version: 2.0
+
+jobs:
+  checkout:
+    docker:
+      - image: circleci/ruby:2-node-browsers
```
Comment (dimaip) [inline .circleci/config.yml]: Why ruby?

Comment (johannessteu) [reply .circleci/config.yml]: It's one of circle ci's ready to use images that got the requirements we need installed and spin up fast

Comment (johannessteu) [reply .circleci/config.yml]: You could probably use any of those with `node-browsers` on it
https://circleci.com/docs/2.0/circleci-images/

Comment (dimaip) [reply .circleci/config.yml]: Ah, I see, will switch it later

**Pull request closed without merging**